RecoTracker/ConversionSeedGenerators/python/ConversionStep2_cff.py

```diff
@@ -197,48 +197,46 @@
 # QUALITY CUTS DURING TRACK BUILDING
 import TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff
 conv2CkfTrajectoryFilter = TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff.CkfBaseTrajectoryFilter_block.clone(
-        maxLostHits = 1,
-        minimumNumberOfHits = 3,
-        minPt = 0.1
-    )
+    maxLostHits         = 1,
+    minimumNumberOfHits = 3,
+    minPt               = 0.1
+)
 
 # TRACK BUILDING
 import RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi
 conv2CkfTrajectoryBuilder = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilder.clone(
     trajectoryFilter = cms.PSet(refToPSet_ = cms.string('conv2CkfTrajectoryFilter')),
```

Comment:
This type can also get dropped

Reply:
@perrotta I've tried to drop this type in refToPSet_ in this ConversionStep2_cff.py file as you commented but It turned out making a TypeError when 'cms.string' was removed  and it also happened at the clustersToSkip in below.

Reply:
Did you try something similar to https://github.com/cms-sw/cmssw/pull/30353/files#diff-d58f48fc8f8fa553fd29f19cd15a5ea9R52 ?